modules/core/src/lapack.cpp

```diff
@@ -277,19 +277,19 @@ template<typename T> struct VBLAS
     int givensx(T*, T*, int, T, T, T*, T*) const { return 0; }
 };
 
-#if CV_SIMD
+#if (CV_SIMD || CV_SIMD_SCALABLE)
```

Comment:
Let's also disable this block for now. Without it core tests pass for me (built with GCC 13.2).